lighthouse-core/report/v2/templates.html

```diff
@@ -0,0 +1,26 @@
+<!-- Lighthouse category score -->
+<template id="tmpl-lighthouse-category-score">
+  <div class="lighthouse-score">
+    <div class="lighthouse-score__value"><!-- fill me --></div>
+    <div class="lighthouse-score__header">
+      <div class="lighthouse-score__snippet">
+        <span class="lighthouse-score__title"><!-- fill me --></span>
+      </div>
+      <div class="lighthouse-score__description"><!-- fill me --></div>
```

Comment:
I like the idea, but right now it's not worth it. Things are going to move around as we iterate. Don't want comments to get out of whack.